x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/cases_webhook/webhook.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { lazy } from 'react';
+import { i18n } from '@kbn/i18n';
+import { ActionTypeModel, GenericValidationResult } from '../../../../types';
+import { CasesWebhookActionParams, CasesWebhookConfig, CasesWebhookSecrets } from './types';
+
+export function getActionType(): ActionTypeModel<
+  CasesWebhookConfig,
+  CasesWebhookSecrets,
+  CasesWebhookActionParams
+> {
+  return {
+    id: '.cases-webhook',
+    iconClass: 'logoWebhook',
+    selectMessage: i18n.translate(
+      'xpack.triggersActionsUI.components.builtinActionTypes.casesWebhookAction.selectMessageText',
+      {
+        defaultMessage: 'Send a request to a Case Management web service.',
+      }
+    ),
+    betaBadgeProps: {
```

Comment:
As the title and the content of the technical preview badges are the same across kibana I think it is better to replace `betaBadgeProps` with `isExperimental: boolean`. The framework, if `isExperimental === true`, will put automatically the `label` and the `tooltipContent` with the same text for all connectors.

Reply:
<img width="725" alt="Screen Shot 2022-07-23 at 8 33 48 AM" src="https://user-images.githubusercontent.com/6935300/180609615-a92a75d9-595e-4507-b3b5-091808cebb8a.png">
Looks like `isExperimental` is not valid for `EuiCard`?

Reply:
I am not saying to pass `isExperimental ` in the `EuiCard` component. I am saying to define the connector as

```
{
    id: '.cases-webhook',
    isExperimental: true,
    ....
}
```

and then in `x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_menu.tsx`, `x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/create_connector_flyout/header.tsx` and whenever else is needed to construct the `betaBadgeProps` before passing it to the `EuiCard`.

As a developer, I set the ` isExperimental` to `true` when I create the connector and the framework pass whatever the `EuiCard` needs.

The proposal is to change this https://github.com/elastic/kibana/pull/131762/files/3dc8e3ee74e51d41ee4e14f503221c0f96d23ea6#diff-e473233cb96adf37e6db4e8d7825401e0e4305fc635d56b78043f351cb38d6ddR197 to `isExperimental: boolean` but the feature is the same (you show the experimental badge).